Server/rainbow-server/Sources/Application/Routes/RouteAutheticator.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  RouteAutheticator.swift
+//  Application
+//
+//  Created by Sanjeev Ghimire on 5/14/18.
+//
+
+import Foundation
+import Credentials
+import CredentialsHTTP
+import LoggerAPI
+
+func setupBasicAuth(app: App) {
+    // Setup a dictionary of users. In general, these would be read in from a database
+    // Encode salted passwords using a one way encoding such as PBKDF2 or bcrypt or etc.
+    // For more information, see https://www.owasp.org/index.php/Password_Storage_Cheat_Sheet
+  
+    /// MARK: this could be fetched from the database. keeping it here for now
+    var userDB: [String: EncodedPassword]?  {
+        do {
+            let dbUser =  [ "watsonml" : try EncodedPassword(withName: "rainbowml", password: "(C0r3MLiPh0neGam3!)", encoding: .PBKDF2)]
```

Comment:
Let's not check this into git. Can we load this from a file that isn't checked in? You can pass the file to us separately.

Reply:
@dokun1  I kept it there for now. I do have a comment there to add it to the database. 
@stevemart it will be just one user just to have a level of security. every mobile client will be using the same user/pwd combo.